task/backend/scheduler_metrics.go

```diff
@@ -59,6 +61,13 @@ func newSchedulerMetrics() *schedulerMetrics {
 			Name:      "claims_active",
 			Help:      "Total number of claims currently held.",
 		}),
+		queueDelta: prometheus.NewSummary(prometheus.SummaryOpts{
+			Namespace:  namespace,
+			Subsystem:  subsystem,
+			Name:       "run_queue_delta",
+			Help:       "The time it task from when a task is due until it is started.",
```

Comment:
`The duration in seconds between a run being due to start and actually starting.`